### DISCUSSION TITLE
Allow facilitator to lock or control access to tabs

### DISCUSSION THREAD

**Discussion by rwgunther** [microsoft/vsts-extension-retrospectives#522] [Ideas]
This Retro board is great for remote teams! Since teams enter information on their own browser, it would be handy if the facilitator could virtually lock team members out of the preceding tabs as they move through the process. For example, when we're finished sharing, I'd like to lock out the Collect tab, forcing everyone to the Group tab. This would be especially helpful if you're obscuring feedback—you _need_ everyone to move to the Group tab. Similarly, when that's done, you want everyone on the Vote tab. And once that's done, you pretty much want to lock it down so that any time someone visits the link you share to the board, they see the final Act view and can't go making post-retro changes. Maybe during the retro you might open things up again, but once it's done, it seems you'd always want people to see the final, sorted results.

**Comment by nigeledw** [comment]
I agree this would be a useful feature for the facilitator to draw everyone's attention at specific points of the retro session particularly if it can coincide with the general countdown timer.

However, we should also ensure people don't lose anything they are working on at the time the facilitator switches tab focus. I personally am quite happy to be adding Kudos and What could be improved on while waiting for the countdown timer of the What went well collection.
Maybe this is where, and what other retro tools allow, each person can predefine items before posting them to the retro board.

**Comment by YogeshKapila** [comment]
Yes, I agree. With everyone accessing the retro page at the same time, for a smooth functioning it is better that everyone works at the same phase. For example, while "Collect" phase is ongoing, no one should be able to "group" or "vote". Some kind of locking mechanism is required for a smooth functioning, specially, when all members are remotely accessing the retrospective.
